deploy/olm-catalog/bundle/manifests/migration.openshift.io_migclusters.yaml

```diff
@@ -47,19 +48,26 @@ spec:
           description: MigClusterSpec defines the desired state of MigCluster
           properties:
             azureResourceGroup:
+              description: AzureResourceGroup is for azure clusters -- it's the resource group that in-cluster volumes use.
               type: string
             caBundle:
+              description: If the migcluster needs SSL verification for connections a user can supply a custom CA bundle.
               format: byte
               type: string
             exposedRegistryPath:
+              description: Stores the path of registry route when using direct migration.
               type: string
             insecure:
+              description: Insecure stores the status of the connection with source cluster. True signifies the connection is insecure.
               type: boolean
             isHostCluster:
+              description: IsHostCluster specifies if the cluster is host (where the controller is installed) or not. This is a required field.
               type: boolean
             refresh:
+              description: Refresh True forces the controller to run a connection test on migcluster.
```

Comment:
While this is true, in a generic sense `refresh: true` just forces a re-reconcile of the migcluster, which is going to run the full suite of validations, not just a connection test.

Reply:
Updated the description.